testing/ltp/0001-Fix-static-struct-warning.patch

```diff
@@ -0,0 +1,39 @@
+From 0916b6a9ecac387cb2e266d76d75ff5ab3fb3440 Mon Sep 17 00:00:00 2001
+From: Brennan Ashton <bashton@brennanashton.com>
+Date: Sat, 10 Apr 2021 15:15:29 -0700
+Subject: [PATCH] Fix static struct warning
+
+---
+ .../open_posix_testsuite/conformance/interfaces/mmap/5-1.c      | 2 +-
+ testcases/open_posix_testsuite/stress/threads/sem_open/s-c1.c   | 2 +-
+ 2 files changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/testcases/open_posix_testsuite/conformance/interfaces/mmap/5-1.c b/testcases/open_posix_testsuite/conformance/interfaces/mmap/5-1.c
+index bcb330da6..ad7b6bb2d 100644
+--- a/testcases/open_posix_testsuite/conformance/interfaces/mmap/5-1.c
++++ b/testcases/open_posix_testsuite/conformance/interfaces/mmap/5-1.c
+@@ -34,7 +34,7 @@
+ #include "posixtest.h"
+ #include "tempfile.h"
+ 
+-static struct testcase {
++struct testcase {
+ 	int prot;
+ 	int flags;
+ };
+diff --git a/testcases/open_posix_testsuite/stress/threads/sem_open/s-c1.c b/testcases/open_posix_testsuite/stress/threads/sem_open/s-c1.c
+index 14ad4ea2c..2133b721a 100644
+--- a/testcases/open_posix_testsuite/stress/threads/sem_open/s-c1.c
++++ b/testcases/open_posix_testsuite/stress/threads/sem_open/s-c1.c
+@@ -399,7 +399,7 @@ int main(int argc, char *argv[])
+  * The function returns 0 when r1 is the best for all cases (latency is constant) and !0 otherwise.
+  */
+ 
+-static struct row {
++struct row {
```

Comment:
let's upstream to ltp?

Reply:
I will submit a patch and see if they are open to it.